### DISCUSSION TITLE
How to use different browserslist configuration for dev/build

### DISCUSSION THREAD

**Discussion by XiongLiding** [quasarframework/quasar#13453] [General - Components / Directives / etc] [answered]
In my project, I have to support chrome 43 which doesn't support async/await. Then after transpile, the code changes a lot, and becomes hard to debug both with or without sourcemap.
So I want to use a modern browserslist for dev, and a lower version for build.
How to achieve it? Or any good solution?

**Comment by metalsadman** [comment, marked as answer]
try in their docs they have https://github.com/browserslist/browserslist#configuring-for-different-environments
